apollo-portal/src/main/resources/static/scripts/controller/UserController.js

```diff
@@ -21,19 +21,99 @@ user_module.controller('UserController',
 function UserController($scope, $window, $translate, toastr, AppUtil, UserService, PermissionService) {
 
     $scope.user = {};
+    $scope.createdUsers = [];
+    $scope.filterUser = [];
+    $scope.status = '1'
+    $scope.showSearchUsernameInput = false
+    $scope.searchKey = ''
+    $scope.changeStatus = changeStatus
+    $scope.toggleUsernameSearchInput = toggleUsernameSearchInput
+    $scope.searchUsers = searchUsers
+    $scope.resetSearchUser = resetSearchUser
 
     initPermission();
 
+    getCreatedUsers();
+
     function initPermission() {
         PermissionService.has_root_permission()
         .then(function (result) {
             $scope.isRootUser = result.hasPermission;
         })
     }
 
+    function getCreatedUsers() {
+        UserService.find_users("",true)
+        .then(function (result) {
+            if (!result || result.length === 0) {
+                return;
+            }
+            $scope.createdUsers = [];
+            $scope.filterUser = [];
+            result.forEach(function (user) {
+                $scope.createdUsers.push(user);
+                $scope.filterUser.push(user);
+            });
+        })
+    }
+
+    function changeStatus(status, user){
+        $scope.status = status
+        $scope.user = {}
+        if (user != null) {
+            $scope.user = {
+                username: user.userId,
+                userDisplayName: user.name,
+                email: user.email,
+                enabled: user.enabled,
+            }
+        }
+    }
+
+    function toggleUsernameSearchInput() {
+        $scope.showSearchUsernameInput = !$scope.showSearchUsernameInput
+    }
+
+    function searchUsers() {
+        $scope.searchKey = $scope.searchKey.toLowerCase();
+        var filterUser = []
+        $scope.createdUsers.forEach(function (item) {
+            var userLoginName = item.userId;
+            if (userLoginName && userLoginName.toLowerCase().indexOf( $scope.searchKey) >= 0) {
+                filterUser.push(item);
+            }
+        });
+        $scope.filterUser = filterUser
+    }
+
+    function resetSearchUser() {
+        $scope.searchKey = ''
+        searchUsers()
+    }
+
+    $scope.changeUserEnabled = function (user) {
+        var newUser={}
+        if (user != null) {
+            newUser = {
+                username: user.userId,
+                userDisplayName: user.name,
+                email: user.email,
+                enabled: 1-user.enabled,
```

Comment:
This logic is correct but I'm afraid it's not easy to understand for future maintainers...
How about:
```suggestion
                enabled: user.enabled === 1 ? 0 : 1,
```